src/components/pages/Tests/TestsList/TestsList.tsx

```diff
@@ -86,7 +88,7 @@ const TestsList: FC = () => {
       queryFilters={filters}
       setQueryFilters={setFilters}
       data={tests}
-      isLoading={isLoading || !isAvailable}
+      isLoading={isLoading}
```

Comment:
Probably previous behavior is expected:
```suggestion
      isLoading={isLoading || !isSystemAvailable}
```